creavision/crvcamera_wdm.cpp

```diff
@@ -78,43 +78,16 @@ void CCameraWDM::DoClose ()
 	}
 }
 
-IplImage *CCameraWDM::DoQueryFrame()
-{
-	if (!CCameraWDM::DoQueryFrame(m_Image)) return NULL;
-
-	return m_Image.ptr();
-}
-
-bool CCameraWDM::DoQueryFrame(CIplImage& image)
+bool CCameraWDM::DoQueryFrame(cv::Mat &frame)
```

Comment:
Are native drivers still relevant or can they be dropped entirely? The driver selection seems to be hard-coded to opencv.

Reply:
Not sure